default/app.yaml

```diff
@@ -0,0 +1,3 @@
+runtime: python37 # or python37 for Python 3.7
+
+instance_class: F1
```

Comment:
Not necessary since this is the default value

Reply:
Fixed